crates/turborepo/src/main.rs

```diff
@@ -1,10 +1,13 @@
-use std::{env::current_exe, process, process::Stdio};
+use std::{env::current_exe, fs::canonicalize as fs_canonicalize, process, process::Stdio};
 
 use anyhow::Result;
+#[cfg(windows)]
+use dunce::canonicalize as fs_canonicalize;
```

Comment:
Looking at [`dunce` source](https://docs.rs/dunce/latest/src/dunce/lib.rs.html#61-72), I think we can get away with just importing `canonicalize` and skip doing the `#[cfg(windows)]` stuff ourselves
```suggestion
use dunce::canonicalize as fs_canonicalize;
```

Reply:
Love it!